src/core/MythosArea.ttslua

```diff
@@ -12,9 +12,7 @@ local ENCOUNTER_DISCARD_AREA = {
   lowerRight = { x = 1.58, z = 0.38 },
 }
 
-local currentScenario
-local useFrontData
-local tokenData
+local currentScenario, useFrontData, tokenData
```

Comment:
Opinions vary on this so up to you, but unless these are tightly related variables one-per-line is usually more readable.  e.g. I might do "local x, y" but not much more than that.

Reply:
Updated